models/tests/test_markov_chain.py

```diff
@@ -56,7 +57,11 @@ def test_multiple_tracks():
 
 
 def test_model_saving_and_loading(tmpdir):
-    path = Path(tmpdir).joinpath('markov_saved.npy')
+
+    if not os.path.exists(tmpdir):
+        os.makedirs(tmpdir, exist_ok=True)
+
+    path = Path(tmpdir).joinpath('markov_saved.npz')
```

Comment:
`.npz` or `.npy`?

Reply:
Npz is correct, it is a collection of `npy`s

Reply:
kk, thanks for the insight :>